packages/lab/stories/slider/slider.stories.tsx

```diff
@@ -1,60 +1,195 @@
-import { Button, Card, Checkbox } from "@salt-ds/core";
-import { Slider, type SliderProps } from "@salt-ds/lab";
+import {
+  FlexLayout,
+  FormField,
+  FormFieldHelperText,
+  FormFieldLabel,
+  Input,
+} from "@salt-ds/core";
+import { Slider, type SliderProps, type SliderValue } from "@salt-ds/lab";
 import type { StoryFn } from "@storybook/react";
+import { type ChangeEvent, useState } from "react";
 
 export default {
   title: "Lab/Slider",
   component: Slider,
 };
 
-const SliderTemplate: StoryFn<SliderProps> = (args) => {
-  return <Slider {...args} />;
+const Template: StoryFn<SliderProps> = ({ ...args }) => {
+  return <Slider style={{ width: "300px" }} {...args} />;
 };
 
-const SliderOnACardTemplate: StoryFn<SliderProps> = (props) => {
+export const Default = Template.bind({});
+Default.args = {
+  min: 0,
+  max: 10,
+  "aria-label": "default",
+};
+
+export const NonZeroInput = Template.bind({});
```

Comment:
Not sure what's "non zero" mean here? i can input 0...?

Reply:
Yep, agreed. Have changes to "With Negative Bounds" to reflect what the story is actually showing.